base_locale_uom_default/models/res_lang.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ResLang(models.Model):
+
+    _inherit = 'res.lang'
+
+    default_uom_ids = fields.Many2many(
+        string='Default Units',
+        comodel_name='product.uom',
+    )
+
+    @api.multi
+    @api.constrains('default_uom_ids')
+    def _check_default_uom_ids(self):
+        for record in self:
+            uoms = {}
```

Comment:
I thought about that- but it's two iterations, two data structures, and one conditional vs. one iteration and one data structure and no conditional. It seemed more efficient to do this direction IMO, although I guess there's not many records so it probably doesn't make much of a difference.